angular-app/src/app/models/interfaces/IAvailability.ts

```diff
@@ -0,0 +1,20 @@
+import { IUser } from './IUser';
+
+export enum DaysOfWeek {
+    'MONDAY' = 'Monday',
+    'TUESDAY' = "Tuesday",
+    'WEDNESDAY' = "Wednesday",
+    'THURSDAY' = "Thursday",
+    'FRIDAY' = "Friday", 
+    'SATURDAY' = "Saturday",
+    'SUNDAY' = "Sunday",
+}
+
+export interface IAvailability {
```

Comment:
These data formats probably make more sense to the end user than the ones the backend is using, so you should probably keep using these. We'll need to convert from your "easy for user" model to my "easy for program" model before posting to my schedule endpoints. This is a good practice, as it doesn't lock us into using a single representation of the idea of a schedule when that would be detrimental to at least one place where it's used. Looks good!